<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiPanel</class>
 <widget class="QWidget" name="GuiPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>504</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="meshData">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Data</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="boxMaterial">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Material:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="boxNVerts">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>130</y>
      <width>42</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="number of vertices"/>
    </property>
    <property name="numDigits">
     <number>4</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxFlags">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="flags"/>
    </property>
    <property name="inputMask">
     <string>000000000000; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Flags:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="boxTexture">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="diffuse texture name (depends on material)"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Texture:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>110</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>verts:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="boxNFaces">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>130</y>
      <width>42</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="number of faces"/>
    </property>
    <property name="numDigits">
     <number>4</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>faces:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="boxNPos">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>130</y>
      <width>42</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="number of vertex positions"/>
    </property>
    <property name="numDigits">
     <number>4</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>pos:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="boxNFrames">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>130</y>
      <width>42</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="number of frames (vertex animation)"/>
    </property>
    <property name="numDigits">
     <number>4</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>frames:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="meshDataAni" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Time of frame</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="frameNumber">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>7</y>
       <width>40</width>
       <height>22</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="boxFlags_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>7</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="flags"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
   </widget>
   <zorder>boxMaterial</zorder>
   <zorder>label_2</zorder>
   <zorder>boxNVerts</zorder>
   <zorder>boxFlags</zorder>
   <zorder>label_3</zorder>
   <zorder>boxTexture</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>boxNFaces</zorder>
   <zorder>label_8</zorder>
   <zorder>boxNPos</zorder>
   <zorder>label_10</zorder>
   <zorder>boxNFrames</zorder>
   <zorder>label_11</zorder>
   <zorder>meshDataAni</zorder>
   <zorder>meshView</zorder>
  </widget>
  <widget class="QGroupBox" name="meshView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>180</width>
     <height>141</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>View</string>
   </property>
   <widget class="QCheckBox" name="cbTexture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>72</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Texture</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbWireframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>72</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>WireFrame</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbVertexcolor">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>84</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertex Color</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbRiggingcolor">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>84</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Rigging Color</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbNocolor">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>84</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>No Color</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbLighting">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>72</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Lighting</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="meshViewAni" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buPause">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>||</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buPlay">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buStop">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[]</string>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>meshData</zorder>
  <zorder>meshView</zorder>
  <zorder>rbRiggingcolor</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
